nise/generators/ocp/ocp_generator.py

```diff
@@ -323,8 +370,42 @@ def _gen_pods(self, namespaces):
                         "mem_request_gig": mem_request_gig,
                         "mem_limit_gig": mem_limit_gig,
                         "pod_labels": self._gen_openshift_labels(),
+
+
                     }
-        return pods, namespace2pod
+                    ros_ocp_data_pods[pod] = {
+                        "namespace": namespace,
+                        "node": node.get("name"),
+                        "resource_id": node.get("resource_id"),
+                        "pod": pod,
+                        "container_name":pod,
+                        "cpu_request_container_avg": randint(75, 100),
+                        "cpu_request_container_sum": randint(25, 75),
+                        "cpu_limit_container_avg": randint(25, 75),
+                        "cpu_limit_container_sum": randint(25, 75),
+                        "cpu_usage_container_avg": randint(25, 75),
+                        "cpu_usage_container_min": randint(0, 25),
+                        "cpu_usage_container_max": randint(75, 100),
+                        "cpu_usage_container_sum": randint(25, 75),
+                        "cpu_throttle_container_avg": randint(25, 75),
+                        "cpu_throttle_container_max": randint(75, 100),
+                        "cpu_throttle_container_sum": randint(25, 75),
+                        "memory_request_container_avg": randint(25, 75),
+                        "memory_request_container_sum": randint(25, 75),
+                        "memory_limit_container_avg": randint(25, 75),
+                        "memory_limit_container_sum": randint(25, 75),
+                        "memory_usage_container_avg": randint(25, 75),
+                        "memory_usage_container_min": randint(0, 25),
+                        "memory_usage_container_max": randint(75, 100),
+                        "memory_usage_container_sum": randint(25, 75),
+                        "memory_rss_usage_container_avg": randint(25, 75),
+                        "memory_rss_usage_container_min": randint(0, 25),
+                        "memory_rss_usage_container_max": randint(75, 100),
+                        "memory_rss_usage_container_sum": randint(25, 75),
+
+                    }
```

Comment:
I believe all of this needs to be included in the `if node.get("namespaces"):` branch above too.

Reply:
done